config_test.yaml

```diff
@@ -36,6 +48,8 @@ proxy:
   data:
     shuffle: True
     seed : 10
+    norm: False
+  dropout_samples: 20
```

Comment:
dropout_samples has to do with the acquisition function for uncertainty rather the proxy in itself, not really important though

Reply:
Agreed. I could change that in this PR or we could create a separate PR for config after we discuss how we want to go about it for diff envs